rep-2005.rst

```diff
@@ -0,0 +1,237 @@
+With the curated list below we aim to provide community members with guidance regarding which parts of the ROS 2 ecosystem are widely used, actively maintained, and of high quality.
+Furthermore, we encourage contributors to focus their efforts on items from this list.
+
+
+Scope
+=====
+
+The list below is curated to include repositories that are:
+
+* open source (not proprietary);
+* connected to the ROS 2 project (not general purpose software, except when included with ROS 2-specific bindings);
+* broadly usable (not narrowly tailored for uncommon use cases); and
+* evidently used by a nontrivial part of the community.
+
+We aim that all of these repositories are also:
+
+* actively maintained and following accepted software development processes, including code review and testing.
+
+
+Change process
+==============
+
+This content is maintained at GitHub in the [REP repo](https://github.com/ros-infrastructure/rep).
+To propose changes, such as adding or removing items, make a pull request to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
+
+
+Content
+=======
+
+* Buildsystem, package index and tooling around
+
+  * `ament/ament_cmake <https://github.com/ament/ament_cmake>`_
+  * `ament/ament_index <https://github.com/ament/ament_index>`_
+  * `ament/ament_package <https://github.com/ament/ament_package>`_
+  * `ament/ament_lint <https://github.com/ament/ament_lint>`_
+  * `ros/ros_environment <https://github.com/ros/ros_environment>`_
+  * `ros-infrastructure/bloom <https://github.com/ros-infrastructure/bloom>`_
+  * `ros-infrastructure/catkin_pkg <https://github.com/ros-infrastructure/catkin_pkg>`_
+  * `ros-infrastructure/rosdep <https://github.com/ros-infrastructure/rosdep>`_
+  * `ros-infrastructure/rosdistro <https://github.com/ros-infrastructure/rosdistro>`_
+  * `ros-infrastructure/ros_buildfarm <https://github.com/ros-infrastructure/ros_buildfarm>`_
+  * `ros2/ament_cmake_ros <https://github.com/ros2/ament_cmake_ros>`_
+
+* Third party packages
+
+  * `eProsima/foonathan_memory_vendor <https://github.com/eProsima/foonathan_memory_vendor>`_
+  * `ament/googletest <https://github.com/ament/googletest>`_
+  * `ament/uncrustify_vendor <https://github.com/ament/uncrustify_vendor>`_
+  * `ros2/console_bridge_vendor <https://github.com/ros2/console_bridge_vendor>`_
+  * `ros2/poco_vendor <https://github.com/ros2/poco_vendor>`_
+  * `ros2/spdlog_vendor <https://github.com/ros2/spdlog_vendor>`_
+  * `ros2/tinydir_vendor <https://github.com/ros2/tinydir_vendor>`_
+  * `ros2/tinyxml_vendor <https://github.com/ros2/tinyxml_vendor>`_
+  * `ros2/tinyxml2_vendor <https://github.com/ros2/tinyxml2_vendor>`_
+  * `ros2/yaml_cpp_vendor <https://github.com/ros2/yaml_cpp_vendor>`_
```

Comment:
What about `sqlite_vendor` and `zstd_vendor` for `rosbag2`? Should those be considered ROS2 Third party or limited to `rosbag2` only?

Reply:
Both packages are part of the `rosbag2` repository which is already part of the list.